app/config/services.yml

```diff
@@ -18,6 +18,12 @@ services:
         tags:
             - { name: twig.extension }
 
+    app.twig.locale_extension:
+        class:     AppBundle\Twig\LocaleExtension
+        arguments: ["@service_container"]
```

Comment:
We can't inject the entire service container because that's a practice we always discourage. Injecting just the `%locales%` parameters will be enough.